CMakeLists.txt

```diff
@@ -8,6 +8,6 @@ cmake_minimum_required(VERSION 3.5)
 
 project(torcharrow)
 
-# TODO: Move global build config from csrc/velox/CMakeLists.txt to here
+find_package(Torch REQUIRED)
```

Comment:
@wenleix Is PyTorch going to be a mandatory dependency of TorchArrow?

Reply:
@mthrok Ideally we want to keep PyTorch as a soft-dep for TorchArrow.